dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -60,7 +78,7 @@ internal object CoreFeature {
         setupInfoProviders(appContext)
 
         setupOkHttpClient(needsClearTextHttp)
-
+        dataUploadScheduledExecutor = ScheduledThreadPoolExecutor(1)
```

Comment:
why do you recreate an instance here when it's already initialized in `init` ? 
Also if this is needed, why only initialize `dataUploadScheduledExecutor` and not `dataPersistenceExecutorService`

Reply:
Left over from previous trials :(. Will fix that.

Reply:
Also, this `1` could be moved to a meaningful constant 🙌 .